packages/angular/src/generators/init/init.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Tree,
+  formatFiles,
+  convertNxGenerator,
+  GeneratorCallback,
+} from '@nrwl/devkit';
+import { jestInitGenerator } from '@nrwl/jest';
+import { angularInitGenerator as nxAngularInitGenerator } from '@nrwl/angular/generators';
+import { viteInitGenerator } from '@nxext/vite';
+import { setDefaultCollection } from '@nrwl/workspace/src/utilities/set-default-collection';
+
+import { Schema } from './schema';
+import { UnitTestRunner } from '@nrwl/angular/src/utils/test-runners';
+
+function normalizeOptions(schema: Schema) {
+  return {
+    ...schema,
+    unitTestRunner: schema.unitTestRunner ?? 'jest',
+  };
+}
+
+export async function angularInitGenerator(tree: Tree, schema: Schema) {
+  const options = normalizeOptions(schema);
+
+  let jestInstall: GeneratorCallback;
+  if (options.unitTestRunner === 'jest') {
+    jestInstall = await jestInitGenerator(tree, {});
+  }
+
+  await nxAngularInitGenerator(tree, {
+    ...options,
+    unitTestRunner: UnitTestRunner.Jest,
+  }).then(() => setDefaultCollection(tree, '@nxext/angular'));
```

Comment:
The then isn't necessary

Reply:
Should work with the setDefault as just a normal call after nxAngularInitGenerator